si.go

```diff
@@ -68,7 +68,7 @@ func ComputeSI(input float64) (float64, string) {
 	value := mag / math.Pow(10, exponent)
 
 	// Handle special case where value is exactly 1000.0
-	// Should return 1M instead of 1000k
+	// Should return 1 M instead of 1000 k
```

Comment:
Looking at this again, I realize these are not units but rather multipliers, and I probably shouldn't have touched them. Oh well.